frontend/src/employees/Employees.tsx

```diff
@@ -11,17 +13,38 @@ type EmployeesProps = {
 
 export default function Employees(props: EmployeesProps) {
 
-    return(
+    const [filterValue, setFilterValue] = useState<string>("");
```

Comment:
Man bräuchte in diesem Fall `<string>` nicht, weil es durch den initialvalue schon gesetzt wird.

Reply:
Leaving it for Florian's happiness ;)